### DISCUSSION TITLE
Lending a hand... for ARM?

### DISCUSSION THREAD

**Discussion by iamhumanipromise** [kth5/archpower#57] [Ideas]
The Arch ARM folks got rid of Raspberry Pi 1 and Pi Zero support awhile back with its their ARMv6 CPU. 

**Seeing if there is a home here for another lost RISC CPU!** @kth5: your genius keeps me coming back here and wishing I could help more 

Can provide a Pi 1B+ and some Zero boards if needed! (both have 512MB RAM,  Broadcom VideoCore IV with H264 support and OpenGL ES 2.0; so there is some life left) 

And can compile stuff if you need me to set up a buildmachine for you on its own VLAN so you can SSH into it.

**Comment by kth5** [comment]
Risking to necro-bump:

I'm not working on anything ARM or x86* just open ISAs (w/ `powerpc64` and `powerpc` being somewhat a by-product).

I doubt you'd have fun building software on such a low-end SBC as the Raspberry Pi range if there's the option to cross compile. But thank you for the offer. :)
